composer.json

```diff
@@ -22,10 +22,11 @@
     "source": "https://github.com/bryanjhv/slim-session.git"
   },
   "require": {
+    "php": ">=8",
     "slim/slim": "^4.0"
   },
   "require-dev": {
-    "phpunit/phpunit": "^7.0"
+    "phpunit/phpunit": "^9.5"
```

Comment:
Does this version support PHP 5?

Reply:
No. If need active support with PHP 5.x. use old PhpUnit version.
(But will be not work with PHP8 or newer)